google-cloud-speech/lib/google/cloud/speech/v1beta1/speech_client.rb

```diff
@@ -82,17 +79,40 @@ def initialize \
               scopes: ALL_SCOPES,
               client_config: {},
               timeout: DEFAULT_TIMEOUT,
-              app_name: "gax",
-              app_version: Google::Gax::VERSION
+              app_name: nil,
+              app_version: nil,
+              lib_name: nil,
+              lib_version: ""
             # These require statements are intentionally placed here to initialize
             # the gRPC module only when it's required.
             # See https://github.com/googleapis/toolkit/issues/446
             require "google/gax/grpc"
             require "google/cloud/speech/v1beta1/cloud_speech_services_pb"
 
-            google_api_client = "#{app_name}/#{app_version} " \
-              "#{CODE_GEN_NAME_VERSION} gax/#{Google::Gax::VERSION} " \
-              "ruby/#{RUBY_VERSION}".freeze
+            @operations_client = Google::Longrunning::OperationsClient.new(
+              service_path: service_path,
+              port: port,
+              channel: channel,
+              chan_creds: chan_creds,
+              scopes: scopes,
+              client_config: client_config,
+              timeout: timeout,
+              app_name: app_name,
+              app_version: app_version,
+              lib_name: lib_name,
```

Comment:
Yes, it is covered by the dependency, but we need to bump the minimum to 0.8.1, since 0.8.0 will error.